### DISCUSSION TITLE
Is it possible to restart the camera without rebooting?

### DISCUSSION THREAD

**Discussion by Djelibeybi** [X1Plus/X1Plus#406] [Q&A]
There is a known issue wherein only the first RTSP connection to the camera is stable. If that connection drops, you cannot reconnect to the camera via RTSP without restarting the X1C. This issue occurs on both the stock firmware and X1Plus.

I'm wondering if it's possible to restart the webcam process via SSH or MQTT without having to restart the entire printer? That would make it possible to trigger if the RTSP stream fails.

**Comment by markfrancisonly** [comment]
Or better yet, patch the RTSP server to automatically restart after a crash

**Comment by jngmrks** [comment]
I would prefer an RTSP server without RTSPS. Just RTSP, username, password and that's it. The whole thing would probably run faster and without major crashes.

**Reply by 3D-Pink-Girl** [reply]
How do you get the rtsps url please? I'm trying to get live view on VLC. So I can keep eye on my X1C on my android phone. Thanks

**Comment by Djelibeybi** [comment]
So, you can kill the camera process by running the following via SSH:

```bash
kill -9 `pidof ipcam`
```

And then restart it on the touchscreen or via VNC by going to Settings -> General -> LAN Access and toggling RTSP server back on (it will toggle itself off a few seconds after the command above is run). 

If anyone knows how to do the second thing via SSH, we'll have something resembling a workaround, I guess. :)
